doc/architecture/backend.md

```diff
@@ -0,0 +1,56 @@
+# Backend Architecture
+
+## Structure
+
+The backend is split using a 3-layer architecture with controller, service and persistence layer.  
+Controller and service layer can be found in their respective folders.  
+The persistence layer is currently part of `model/entity/`.
+
+### Controller
+
+The controller layer contains all endpoints of the application.  
```

Comment:
Please first clarify what endpoints are in our context.

Reply:
I will do it in the glossary.